2-mandatory/4-stocks.js

```diff
@@ -65,9 +85,19 @@ function getPriceChanges(closingPricesForAllStocks) {
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
     // TODO
+   highestPrice =[];
+   for(let i=0; i < closingPricesForAllStocks.length; i++) {
+     let highestNum = closingPricesForAllStocks[i].sort(function(a,b){
+         return b-a;} )[0];
+         highestPrice.push(
+           "The highest price of " + stocks[i].toUpperCase() + " " + "in the last 5 days was " + highestNum.toFixed(2));
+
+   }
+   return highestPrice;
```

Comment:
nice solution :) The `Math.max` method is useful for finding the highest number, you can pass in an array like so:
let highestNum = Math.max(...closingPricesForAllStocks[i])